content/en/user-guide/chaos-engineering/chaos-experiments/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Chaos Experiments"
+linkTitle: "Chaos Experiments"
+description: Chaos Experiments allows users to conduct chaos engineering experiments and test system reliability by injecting stress or failure in some part of a system.
+tags: ["Enterprise plan"]
+aliases:
+  - /user-guide/chaos-engineering/web-application-dashboard/
+  - /user-guide/chaos-engineering/chaos-application-dashboard/
```

Comment:
**note**: Added this extra alias—if I understand correctly, that's how we manage redirects. 🤷